Library/Source/Console.h

```diff
@@ -8,23 +8,17 @@ namespace babylon
     class Console final : public Napi::ObjectWrap<Console>
```

Comment:
Is it possible to log to console from C++?

Reply:
Can we just add a ref to the LogCallBack in RuntimeImpl and call it from NativeEngine m_runtimeImpl.m_logCallBack()...)

Reply:
We could do that, but I'm not sure I understand why we would. In my mind at least, the `NativeEngine`'s status as a C++ object is incidental; philosophically, it really _belongs_ to JavaScript. The JavaScript's approach to logging is through the `Console` object, so by default I'd expect the `NativeEngine`'s C++ implementation to do so as well. Would going through the `Console` have a problem that going through `RuntimeImpl` would avoid?

Reply:
I'm torn on this. While I agree the Console class is intended for JavaScript, going through JavaScript is quite a bit more round about. How about we add logging as a feature to the Runtime contract and then Console can just call the Runtime?

Reply:
I see. One way to do it is to extend the return values from native function. For example, the loadTexture function returns a bool telling if it succeeded or not. More enums are needed like mipmap wanted but not generated, ...
Also possible, a GetLastError function in the contract (a la OpenGL) to get more infos on a failing function call.